README.md

```diff
@@ -18,47 +8,67 @@ You can check it on [Online DEMO](https://sveltejs.github.io/svelte-eslint-parse
 [![Build Status](https://github.com/sveltejs/svelte-eslint-parser/workflows/CI/badge.svg?branch=main)](https://github.com/sveltejs/svelte-eslint-parser/actions?query=workflow%3ACI)
 [![Coverage Status](https://coveralls.io/repos/github/sveltejs/svelte-eslint-parser/badge.svg?branch=main)](https://coveralls.io/github/sveltejs/svelte-eslint-parser?branch=main)
 
-## ⤴️ Motivation
+<div align="center">
+
+# svelte-eslint-parser
+
+## [Svelte](https://svelte.dev/) parser for [ESLint](https://eslint.org/).
+
+[Live DEMO](https://sveltejs.github.io/svelte-eslint-parser/playground) •
+[Discord](https://svelte.dev/chat)
+
+</div>
 
-The [svelte-eslint-parser] aims to make it easy to create your own ESLint rules for [Svelte].
+## Motivation
 
-The [`eslint-plugin-svelte`] is an ESLint plugin that uses the [svelte-eslint-parser]. I have already [implemented some rules].
+The `svelte-eslint-parser` aims to make it easy to create your own ESLint rules for [Svelte](https://svelte.dev/).
 
-[`eslint-plugin-svelte`]: https://github.com/sveltejs/eslint-plugin-svelte
-[implemented some rules]: https://sveltejs.github.io/eslint-plugin-svelte/rules/
+[eslint-plugin-svelte](https://github.com/sveltejs/eslint-plugin-svelte) is an ESLint plugin built upon this parser, and it already [implements some rules](https://sveltejs.github.io/eslint-plugin-svelte/rules/).
 
 ### ESLint Plugins Using svelte-eslint-parser
 
 #### [eslint-plugin-svelte](https://sveltejs.github.io/eslint-plugin-svelte/)
 
 ESLint plugin for Svelte.  
-It provides many unique check rules by using the template AST.
+Provides a variety of template-based checks using the Svelte AST.
 
 #### [@intlify/eslint-plugin-svelte](https://github.com/intlify/eslint-plugin-svelte)
 
-ESLint plugin for internationalization (i18n) with Svelte.  
-It provides rules to help internationalization your application created with Svelte.
+ESLint plugin for internationalization (i18n) in Svelte applications, offering helpful i18n-related rules.
 
-## 💿 Installation
+---
+
+## Installation
 
 ```bash
 npm install --save-dev eslint svelte-eslint-parser
 ```
 
-## 📖 Usage
+---
+
+## Usage
 
-1. Write `parser` option into your ESLint Config file.
-2. Use glob patterns or `--ext .svelte` CLI option.
+1. In your ESLint config, set the `parser` option to `svelte-eslint-parser`.
+2. Use glob patterns or the `--ext .svelte` CLI option to include `.svelte` files.
```

Comment:
Since the `--ext` option cannot be used in a flat config, it seems necessary to remove the related sentence.
https://eslint.org/docs/latest/use/command-line-interface#--ext

Reply:
Oh... THank you! fixed: https://github.com/sveltejs/svelte-eslint-parser/pull/665/commits/b679b4dbe0c906d08e09cf153841036933ec2d26